index.md

```diff
@@ -73,7 +75,7 @@ I use this method along with…
 
 #### Asana
 
-I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) around a year ago, and have been using it on and off for long-term planning of my activities.
+I was introduced to [Asana](https://asana.com/) by a [friend](https://twitter.com/neuroamanda) and have been using it on and off for long-term planning of my activities.
 As a PhD student, you will often need to plan well into the future, with many external deadlines looming ahead: progress reports, vivas, conference submissions etc.
```

Comment:
May want to say something about collaborative uses of Asana (and, related, Trello). As in, if you're dealing with non-tech savvy team members, it may actually hinder progress. Otherwise can be very useful collaborative tool (tasks, sharing files, milestones, record-keeping...)